api/turing/it/database/database.go

```diff
@@ -40,15 +40,15 @@ func migrate(db *sql.DB, dbName string) (*sql.DB, error) {
 
 	if migrations, err := gomigrate.NewWithDatabaseInstance(fmt.Sprintf("file://%s", migrationsFolder),
 		dbName, driver); err != nil {
-		return nil, err
+		return db, err
```

Comment:
What is the reason behind this change?

Reply:
In some scenarios, it was possible that this snippet will throw nil pointer. For instance, if the migration fails (because of typo in the migration file), this line will return `nil` and then there will be a panic at https://github.com/gojek/turing/blob/b67c48b7e1b1777f89ec6c6860e3965c516fff9f/api/turing/it/database/database.go#L77, when `.Close()` is called on `nil`